ERCS/erc-7950.md

```diff
@@ -0,0 +1,400 @@
+---
+eip: ....
```

Comment:
```suggestion
eip: 7554
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Updated to 7554 as well as filename